src/templates/new_request_page.hbs

```diff
@@ -8,6 +8,9 @@
       <a href="/hc/en-us/articles/4409892315291-Find-my-Account-ID-Number">Learn
       how to find your account ID number.</a>
     </p>
+    <p>
+      <b>Please never include your full license key in your message. We will only need the first 4 characters when applicable for troubleshooting.</b>
```

Comment:
`<strong>` instead of `<b>` would be [more appropriate](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong) here